security/access_control.rst

```diff
@@ -156,8 +156,13 @@ options:
 
 * ``roles`` If the user does not have the given role, then access is denied
   (internally, an :class:`Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException`
-  is thrown). If this value is an array of multiple roles, the user must have
-  at least one of them.
+  is thrown). If this value is an array of multiple roles, the user must have:
+  
+  * at least one of them when using the default ``affirmative`` strategy.
+  * more granted than denied roles when using the ``consensus`` strategy.
```

Comment:
Depending on the config it could also be "at least as many granted as denied", but that's probably a bit pedantic :D

Reply:
I know, so revised `affirmative` description in https://github.com/symfony/symfony-docs/blob/4.4/components/security/authorization.rst#access-decision-manager